src/Plugins/ComplexCore/src/ComplexCore/Filters/FillBadDataFilter.cpp

```diff
@@ -0,0 +1,134 @@
+#include "FillBadDataFilter.hpp"
```

Comment:
Review commented out code in parameters, prefilght, and values struct in algo header and leave an explanation or remove it